docs/cunumeric/source/user/notebooks/stencil.ipynb

```diff
@@ -0,0 +1,221 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "G = make_grid(100)\n",
```

Comment:
the 100x100 plot had too much green and it was hard to distinguish the boundary. Maybe smaller?

Reply:
smaller grid size? 100x100 already seems unimpressively small TBH, maybe we could just omit the first plot (or even both plots... I like to see visual things but if they are not actually useful they can be removed)

Reply:
Yeah, I was thinking to show a 10x10 plot and then run the 100x100 example. On second look, I think the plot is fine. Maybe just more clarification in the text.

"As we can see in the plot, the initial conditions have an absolute zero boundary top, left, and right (dark blue) with a higher boundary value on the bottom (yellow) with an intermediate value (green) in the interior."

"After running, we can see the boundary conditions are preserved, but the interior values have adjusted to average values in-between the boundaries."

Not sure what the standard is, but the input values were a bit weird to me. Instead of specifying the absolute zero temperature boundaries as 0, they are given as -273.15.

Reply:
Also I guess FWIW the boundary does seem noticeable to me as it is 

![image](https://user-images.githubusercontent.com/1078448/220425412-1fd4b2be-531b-461b-8df6-bf9adf176ed7.png)

Reply:
@manopapad any preferences here?